experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl

```diff
@@ -131,7 +133,8 @@ end
 radiation = ClimaLSM.PrescribedRadiativeFluxes(
     FT,
     SW_IN_spline,
-    LW_IN_spline;
+    LW_IN_spline,
+    UTC_DATETIME[1];
```

Comment:
we might change this function signature to take in the reference time as an argument, and then we should be able to pass the time in when it is called in update_aux by accessing the reference time from atmos or radiation.
https://github.com/CliMA/ClimaLSM.jl/blob/499bbaefc58eba0b966648ddd92f585604443244/experiments/integrated/ozark/ozark_met_drivers_FLUXNET.jl#L112

Keeping the reference time in UTC (rather than local) is a good idea! I'm not sure which we are using right now? in zenith_angle it looks like local time.

Reply:
We should be using UTC everywhere... in `zenith_angle` it is time in UTC. The simulations starts from 12AM on Jan 1 local, which is UTC + 6 so `DateTime("2005-01-01-06", "yyyy-mm-dd-HH")` gives us UTC.

This is a good idea about changing the function signature and should be easy to implement I will change that.

Reply:
ok! sounds great, thank you!